src/common/expression/ContainerExpression.h

```diff
@@ -63,7 +66,16 @@ class MapItemList final {
   std::vector<Pair> items_;
 };
 
-class ListExpression final : public Expression {
+class ContainerExpression : public Expression {
```

Comment:
Just minor refactoring to provide some generic polymorphic methods for ContainerExpression.